src/nuklear_internal.h

```diff
@@ -1,6 +1,8 @@
 #ifndef NK_INTERNAL_H
 #define NK_INTERNAL_H
 
+#include "nuklear.h"
```

Comment:
Is this a typo or am I missing something?

Reply:
Many definitions in `nuklear_internal.h` depends on `nuklear.h`. Maybe it is not supposed to be parsed by from IDE intellisense. I tried with qt-creator and it couldn't parse it.

Reply:
The point is, that the sources in `src/` are first being amalgamated (concatenated) into `nuklear.h` using `src/paq.sh` (or `src/paq.bat` on Windows). So you must not do anything like this. See the reviewers guide for an overview.

Reply:
I know there is two way to use Nuklear: header only `nuklear.h` or multiple source files `src/nuklear.h`. I understand that Nuklear is only supposed to be used as header only but for me it is easier to navigate around code with the source files than header only `nuklear.h`.

Reply:
To clarify:
a) There is only one way to use Nuklear - namely the `nuklear.h` header.
b) For the sole purpose of **development of Nuklear itself**, the sources are split and maintained in `src/`.

So you absolutely can not use for your app anything in `src/`. Nuklear is a **single header** library, nothing else :wink:.

Reply:
I like header only files but Nuklear is too large for my taste. If Nuklear were split into multiple header only files I would use them instead.

Reply:
@keharriso ok, it's up to you. Look at `src/paq.sh` how it's being amalgamated and try to amalgamate your own small version. If you'll come up with something very simple, don't hesitate and share your work with us as it might be something to contemplate for the future.

Btw. have you seen https://github.com/rxi/microui ?

Reply:
No. Thanks for the tip. I have to think about this.